src/ResourceManagement/MachineLearning/Microsoft.Azure.Management.MachineLearning/generate.cmd

```diff
@@ -8,7 +8,7 @@ setlocal
 
 set autoRestVersion=0.17.0-Nightly20160824
 
-set webServicesSpecFile="https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-machinelearning/2016-05-01-preview/swagger/webservices.json"
+set webServicesSpecFile="https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-machinelearning/2017-01-01/swagger/webservices.json"
```

Comment:
@NonStatic2014 please use the URL containing the commit Id when the spec gets merged instead of the URL containing the branch